app/views/restaurants/new.html.erb

```diff
@@ -0,0 +1,41 @@
+<div style="margin:0 auto; width: 50%; background-color: grey; border: 2px solid black; text-align: center;" >
+<h1>Add Restaurant</h1>

```

Comment:
Cool, I will change it in next commit. Thnx for letting me know. I just watched some videos on YouTube and tried to give style to new view.